Tasks/GitHubReleaseV0/task.json

```diff
@@ -15,18 +15,18 @@
     "version": {
         "Major": 0,
         "Minor": 0,
-        "Patch": 2
+        "Patch": 3
     },
     "demands": [],
     "minimumAgentVersion": "2.0.0",
     "inputs": [
         {
             "name": "gitHubConnection",
             "type": "connectedService:github:OAuth,PersonalAccessToken",
-            "label": "GitHub Connection",
+            "label": "GitHub connection (OAuth or PAT)",
```

Comment:
Are you planning to port this change to M147?

Reply:
Hi Lova.  I wasn't planning to, but I can if you think it's important to release in the next 2 weeks.

Reply:
We have been hearing couple of support calls around. Since it is a straight forward fix, it would be great if you can port it to at least M147 branch.